third_party/proton/proton/mode.py

```diff
@@ -59,7 +59,7 @@ class BaseMode:
 
 @dataclass(frozen=True)
 class PCSampling(BaseMode):
-    name: str = field(default="pc_sampling", init=False)
+    name: str = field(default="pcsampling", init=False)
```

Comment:
`libproton` does not accept `pc_sampling`: https://github.com/triton-lang/triton/blob/9faa7cd00a9490248387461342b2f89d4a700630/third_party/proton/csrc/lib/Session/Session.cpp#L21